superset/viz.py

```diff
@@ -123,24 +122,6 @@ def run_extra_queries(self):
         """
         pass
 
-    def get_fillna_for_col(self, col):
-        """Returns the value for use as filler for a specific Column.type"""
-        if col:
-            if col.is_string:
-                return ' NULL'
```

Comment:
I think @xrmx added some of this logic. `null`-labeled series creates issues on some visualizations. I believe the leading space has to do with sorting NULL first. We may want to reuse some of the logic I added recently around filtering that replaces `None` by  `<NULL>` and empty stings by `<empty string>`. This should only be applied to dimensions / strings.  Note that it applies only to string columns, thus not "zerofying"  NULL values.

I'm pretty sure that removing this will break many charts.

Reply:
@mistercrunch the `'  NULL'` (with a leading space) is somewhat misleading and is inconsistent with SQL Lab which reports `NULL` (or missing values) as `null`. On an unrelated note SQL Lab doesn't seem to sort `NULL` values correctly and it would be good to ensure that `NULL` values are rendered/sorted consistency throughout the application.